src/interfaces.d.ts

```diff
@@ -35,18 +44,18 @@ export interface HasSortEvent {
 }
 
 export interface HasItem {
-	item: ItemProperties<any>;
+	item: ItemProperties;
 }
 
 export interface HasItems {
-	items: ItemProperties<any>[];
+	items: ItemProperties[];
 }
 
 export interface HasValue {
 	value: string;
 }
 
-export interface ItemProperties<T> {
+export interface ItemProperties<T = any> {
```

Comment:
I think it would be very good if dgrid 2 is able to continue supporting data in the same format as dgrid 1 unless there's a compelling reason not to - is there?

While Dojo's data solutions have evolved over the years, the basic format of a data object has been consistent for years. What is the compelling need to change it?

Also - considering dgrid 2 requires a data provider, are my concerns relevant? Can one store be used for consumers that like the old data format, and also for dgrid, with the differences implemented in the data provider?